plugins/vi-mode/vi-mode.plugin.zsh

```diff
@@ -50,13 +50,16 @@ zle -N zle-keymap-select
 # These "echoti" statements were originally set in lib/key-bindings.zsh
 # Not sure the best way to extend without overriding.
 function zle-line-init() {
+  LAST_VI_KEYMAP=$VI_KEYMAP
```

Comment:
```suggestion
  local prev_vi_keymap
  prev_vi_keymap="${VI_KEYMAP:-}"
```
This avoids warnings on `setopt warn_create_global` or failure on `setopt no_unset`.

Reply:
Good to know, thanks!  (made a separate commit; one additional line had to be changed)

Reply:
Eh, guess the `:-`  in my suggestion is unnecessary since the same var is used without that below.